--- lib/mocha/lib/reporters/base.js	1985-10-26 11:15:00.000000000 +0300
+++ lib/mocha/lib/reporters/base.js	2019-04-10 19:23:57.000000000 +0300
@@ -8,7 +8,7 @@
 
 var tty = require('tty');
 var diff = require('diff');
-var milliseconds = require('ms');
+var milliseconds = function (ms) {return ms > 1000 ? (ms/1000).toFixed (1) + 's' : ms + 'ms'};
 var utils = require('../utils');
 var supportsColor = process.browser ? null : require('supports-color');
 var constants = require('../runner').constants;
